packages/namekit-react/src/components/Identity.tsx

```diff
@@ -12,21 +12,46 @@ import {
 } from "@namehash/nameguard";
 import cc from "classcat";
 
-export interface ProfileLinkConfig {
-  getProfileURL: (address: string) => string;
-  getProfileLink: (address: string, children: ReactNode) => JSX.Element;
+export class ProfileLinkGenerator {
+  private baseURL: string;
+  private name: string;
+
+  constructor(name: string, baseURL: string) {
+    this.name = name;
+    this.baseURL = baseURL;
+  }
+
+  getName(): string {
+    return this.name;
+  }
+
+  getProfileURL(address: string): string {
+    return `${this.baseURL}${address}`;
+  }
 }
 
+export const ENSProfileLink = new ProfileLinkGenerator(
+  "ENS",
+  "https://app.ens.domains/",
+);
+
+const DEFAULT_PROFILE_LINKS = [ENSProfileLink];
+
 interface NameKitConfig {
-  profileLinks?: ProfileLinkConfig;
+  profileLinks?: ProfileLinkGenerator[];
```

Comment:
Similar to the feedback above. Can you help me understand the goal for representing this as an array? In my thinking there should be exactly one, not 0 or many.
```suggestion
  profileLinkGenerator: ProfileLinkGenerator;
```